File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { EmailValidationApi2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await EmailValidationApi2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- EmailValidationApi2 SDK exists test

local sdk = require("email-validation-api2_sdk")

describe("EmailValidationApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
